server/rpc.go

```diff
@@ -350,10 +447,11 @@ func StartTranscodeServer(orch Orchestrator, bind string, mux *http.ServeMux, wo
 
 	glog.Info("Listening for RPC on ", bind)
 	srv := http.Server{
-		Addr:         bind,
-		Handler:      &lp,
-		ReadTimeout:  HTTPTimeout,
-		WriteTimeout: HTTPTimeout,
+		Addr:    bind,
+		Handler: &lp,
+		// XXX doesn't handle streaming RPC well; split remote transcoder RPC?
+		//ReadTimeout:  HTTPTimeout,
+		//WriteTimeout: HTTPTimeout,
```

Comment:
Discussion around this issue: https://github.com/livepeer/go-livepeer/issues/576